src/Server.cpp

```diff
@@ -225,6 +227,10 @@ void Server::clearClient(int fd) {
       it->second.removeOperator(&_clients.at(fd));
     it->second.checkAndremoveClientFromTheChannel(fd);
   }
+  if (wasInaChannel)
+    _clients[fd].receiveMessage(
+      FROM_SERVER + "NOTICE" + " " +
+        "You have been removed from the channel\r\n");
```

Comment:
The message would be better like this:
"You have been removed from channel(s)\r\n"
Because it can be one channel or several channels.